task/pytest/0.2/pytest.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pytest
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
```suggestion
    tekton.dev/pipelines.minVersion: "0.17.0"
```

Reply:
Thanks for the review! Changed the minVersion.